src/_includes/layouts/base.njk

```diff
@@ -71,6 +71,7 @@
 		</script>
 		<script src="/js/chart.js"></script>
 		{%- endif -%}
+    {% include "partials/sa11y.njk" %}
```

Comment:
Adding the if statement to the layout instead of in the partial and adding it to the "local" environment too

```suggestion
{%- if settings.env == "dev" or settings.env == "local" -%}
    {% include "partials/sa11y.njk" %}
{%- endif -%}
```